code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -16,6 +16,9 @@ datum
 		var/list/secondary_results = list()		//additional reagents produced by the reaction
 		var/requires_heating = 0
 
+		var/required_temp = 0

```

Comment:
Should probably have a min/max temperature at some point so you don't just throw in 1000*C of something and call it a day.